go/store/nbs/table_index_test.go

```diff
@@ -102,3 +102,23 @@ func TestMMapIndex(t *testing.T) {
 	assert.Equal(t, idx.TableFileSize(), mmidx.TableFileSize())
 	assert.Equal(t, idx.TotalUncompressedData(), mmidx.TotalUncompressedData())
 }
+
+func TestOnHeapTableIndex_ResolveShortHash(t *testing.T) {
```

Comment:
I'd like to see some more comprehensive testing of the resolve method. Specifically, you should construct your own index with a known set of hashes, and then test resolving short hashes of different lengths.
Check out `nbs/table_writer.go` to construct an index from scratch